packages/web-pkg/src/composables/resources/useLoadPreview.ts

```diff
@@ -10,8 +10,10 @@ import {
 import { FolderViewModeConstants } from '../viewMode'
 import { usePreviewService } from '../previewService'
 import { ProcessorType } from '../../services'
-import { useResourcesStore } from '../piniaStores'
+import {useConfigStore, useResourcesStore, useSpacesStore} from '../piniaStores'
```

Comment:
```suggestion
import { useConfigStore, useResourcesStore, useSpacesStore } from '../piniaStores'
```

Reply:
... please run `pnpm lint --fix` 😅

Reply:
our linter broke down in the mean time ;D